### ISSUE TITLE
Generate personal folders sets regular root folders also as personal

### ISSUE THREAD

**Issue by tehnotronic** [nilsteampassnet/TeamPass#1614]
With latest development code if I choose to generate personal folders as admin, I see that in database those regular folders in root (parent_id=0) are also changed to personal folders. This should not happen and the generation script should only change personal folders (numeric titles in nested_tree table).
![screenshot_60](https://cloud.githubusercontent.com/assets/6266189/21846601/8d3c42ec-d800-11e6-883f-663f8049f0df.png)

**Comment by nilsteampassnet** [comment]
Corrected and committed.

Manually the change is in file sources/admin.queries.php
Replace
```
                // Get an array of all folders
                $folders = $tree->getDescendants($record['id'], false, true, true);
                foreach ($folders as $folder) {
```
by
```
                // Get an array of all folders
                $folders = $tree->getDescendants($data['id'], false, true, true);
                foreach ($folders as $folder) {
```

**Comment by nilsteampassnet** [comment]
Corrected in 2.1.27
